src/pages/studyView/StudyViewPage.tsx

```diff
@@ -174,6 +176,15 @@ export default class StudyViewPage extends React.Component<
                         // Open group comparison manager if there are shared groups in the url
                         this.store.showComparisonGroupUI = true;
                     }
+                } else if (datum.startsWith('sharedCustomData')) {
```

Comment:
Do you want to consider using a library to deal with hash then you can model them? https://github.com/sindresorhus/query-string#readme